nebula-components/components/_progress-bar.scss

```diff
@@ -0,0 +1,12 @@
+@if $nb-use-progress-bar {
+  $nb-color-alpha-light: lighten($nb-color-alpha, 15%);
+
+  .#{$nb-namespace}c-progress-bar {
+    border: 1px solid $nb-color-base-border;
+  }
+
+  .#{$nb-namespace}c-progress-bar__indicator {
+    height: ($nb-spacing-unit-double - $nb-spacing-unit-half);
+    background: repeating-linear-gradient(-45deg, $nb-color-alpha-light, $nb-color-alpha-light $nb-spacing-unit, $nb-color-alpha $nb-spacing-unit, $nb-color-alpha $nb-spacing-unit-double); //linear-gradient(to bottom, $nb-color-grey-v-light, $nb-color-info);
```

Comment:
Just need to remove the comment at the end of the line.

Reply:
Done